packages/lit-html/src/test/parts_test.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * @license
+ * Copyright (c) 2020 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+import {
+  Directive,
+  directive,
+  html,
+  noChange,
+  NodePart,
+  render,
+} from '../lib/lit-html.js';
+import {assert} from '@esm-bundle/chai';
+import {stripExpressionComments} from './test-utils/strip-markers.js';
+import {detachNodePart, restoreNodePart} from '../lib/parts.js';
+
+suite('lit-html', () => {
+  let container: HTMLDivElement;
+
+  setup(() => {
+    container = document.createElement('div');
+  });
+
+  test('detach / restore', () => {
```

Comment:
What this is testing is not super obvious. Maybe add some more comments?

Reply:
Added comment